code/controllers/subsystem/pois.dm

```diff
@@ -0,0 +1,186 @@
+/// Used to specify which POI list has changed when POIs get added or removed from the mob list.
+#define POI_MOBS 1
+/// Used to specify which POI list has changed when POIs get added or removed from the other list.
+#define POI_OTHER 2
+
+/// Subsystem for managing valid POIs that can be orbitted.
+SUBSYSTEM_DEF(pois)
+	name = "POIs"
```

Comment:
```suggestion
	name = "Points of Interest"
```

Seems nitpicky, but I expanded the element name out for a reason, "poi" is fairly internal.

Reply:
I also would be glad if you changed `SSpois` to `SSpoints_of_interest`.

Reply:
Changed